wp-content/plugins/core/src/Routes/Sample_Route.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Sample route.
+ */
+declare( strict_types=1 );
+
+namespace Tribe\Project\Routes;
+
+use Tribe\Libs\Routes\Abstract_Route;
+
+/**
+ * Class to define a sample route.
+ */
+class Sample_Route extends Abstract_Route {
+	/**
+	 * Registers routes.
+	 *
+	 * @return void
+	 */
+	public function register() : void {
+		add_filter( 'core_js_config', [ $this, 'js_config' ] );
```

Comment:
Our instances should never be calling any actions/filters, this should be moved to a `Routes_Subscriber.php` reach out if you need to and we discuss how to best adjust the architecture to accommodate this.